modin/data_management/query_compiler/pandas_query_compiler.py

```diff
@@ -2171,55 +2140,6 @@ def back(self, n):
 
     # End Head/Tail/Front/Back
 
-    # Data Management Methods
-    def free(self):
-        """In the future, this will hopefully trigger a cleanup of this object.
-        """
-        # TODO create a way to clean up this object.
-        return
-
-    # END Data Management Methods
-
-    # To/From Pandas
-    def to_pandas(self):
-        """Converts Modin DataFrame to Pandas DataFrame.
-
-        Returns:
-            Pandas DataFrame of the DataManager.
-        """
-        df = self.data.to_pandas(is_transposed=self._is_transposed)
-        if df.empty:
-            dtype_dict = {
-                col_name: pandas.Series(dtype=self.dtypes[col_name])
-                for col_name in self.columns
-            }
-            df = pandas.DataFrame(dtype_dict, self.index)
-        else:
-            ErrorMessage.catch_bugs_and_request_email(
-                len(df.index) != len(self.index) or len(df.columns) != len(self.columns)
-            )
-            df.index = self.index
-            df.columns = self.columns
-        return df
-
-    @classmethod
-    def from_pandas(cls, df, block_partitions_cls):
```

Comment:
`from_pandas` might be okay as a non-abstract method. It depends on the index/columns implementation

Reply:
I moved it to abstract since I did not want the index/columns to be all the same in case there is some IR that handles index/columns efficiently (i.e., better than pandas)